cloudDetails/gce.json

```diff
@@ -45,44 +41,22 @@
                 }
             },
 
-            "n2-highmem-8": {
-                "roachprodArgs": {
-                    "gce-zones": "us-east1-c",
-                    "west-gce-zones": "us-west1-a"
-                }
-            },
-            "n2-highmem-32": {
-                "roachprodArgs": {
-                    "gce-zones": "us-east1-c",
-                    "west-gce-zones": "us-west1-a"
-                }
-            },
+            "n2-highmem-8": null,
+            "n2-highmem-32": null,
```

Comment:
We don't care about the zones for these instance types?

Reply:
@srosenberg  They'll all just use the default az-s:
```
            "gce-zones": "us-east4-c",
            "west-gce-zones": "us-west1-a",
```
If there's `roachprodArgs` specified under a certain machine type, they will override the default `roachprodArgs` (See [here](https://github.com/cockroachlabs/cloud-report/pull/88/files#diff-b4db402b8cd490eb43f66782faad675f47cd95aa23c394bebe9591ae0cbfa797L87))when creating this machine.

Reply:
yep, I see the override logic here [1]; thanks!

[1] https://github.com/cockroachlabs/cloud-report/blob/master/cmd/generate.go#L447